infra/modules/lightsail/main.tf

```diff
@@ -14,7 +14,27 @@ resource "aws_lightsail_container_service" "garden_service" {
 
 # IAM stuff
 # note: lightsail can't assume iam roles, so we define an iam user here to attach policies to
-# but inject the actual credentials for the app at deployment time with the gh action
+# but inject the actual user credentials for the app at deployment time with the gh action
 resource "aws_iam_user" "lightsail_user" {
   name = "garden_lightsail_user_${var.env}"
 }
+
+data "aws_iam_policy_document" "secrets_policy" {
```

Comment:
I had half a mind to put this in the secretsmanager module, but since it's so tightly coupled to the IAM user dedicated to the lightsail deployments I figured it made just as much sense here. Could be convinced either way

Reply:
I like the conceptual grouping